packages/odyssey-storybook/src/components/odyssey-mui/Button/Button.stories.tsx

```diff
@@ -12,54 +12,163 @@
 
 import type { Meta, StoryObj } from "@storybook/react";
 
-import { Button, AddIcon } from "@okta/odyssey-react-mui";
+import { Button, Icon, iconDictionary } from "@okta/odyssey-react-mui";
 import type { ButtonProps } from "@okta/odyssey-react-mui";
 import { MuiThemeDecorator } from "../../../../.storybook/components/MuiThemeDecorator";
 
 import { userEvent, within } from "@storybook/testing-library";
 import { expect } from "@storybook/jest";
 
+const iconOptions = ["None", ...Object.keys(iconDictionary)];
+const iconMapping = Object.fromEntries(
+  iconOptions.map((value) => [
+    value,
+    value !== "None" ? (
+      <Icon
+        name={value as keyof typeof iconDictionary}
+        label={value}
+        size="medium"
+      />
+    ) : null,
+  ])
+);
+
+type ButtonStoryProps = Omit<ButtonProps, "startIcon" | "endIcon"> & {
+  startIcon: keyof typeof iconDictionary;
+  endIcon: keyof typeof iconDictionary;
+};
```

Comment:
Why are we bringing in all the icons for Button stories?

I think we need this code in other places too, so it might be good to put that in a separate file and import that logic in `Button`.

Reply:
I refactored this to be more consistent with how it's used in production features, and extracted it into a separate file.